source/mig.md

```diff
@@ -0,0 +1,76 @@
+
+### Determine MIG Names
+
+The simplest way to determine the names of MIG instances is to run `nvidia-smi -L` on the command line.
+
+```bash
+$ nvidia-smi -L
+GPU 0: NVIDIA A100-PCIE-40GB (UUID: GPU-84fd49f2-48ad-50e8-9f2e-3bf0dfd47ccb)
+  MIG 2g.10gb     Device  0: (UUID: MIG-41b3359c-e721-56e5-8009-12e5797ed514)
+  MIG 2g.10gb     Device  1: (UUID: MIG-65b79fff-6d3c-5490-a288-b31ec705f310)
+  MIG 2g.10gb     Device  2: (UUID: MIG-c6e2bae8-46d4-5a7e-9a68-c6cf1f680ba0)
+```
+
+In the example case above the system has one NVIDIA A100 with 3 x 10GB MIG instances. In the next sections we will see how to use the instance names to startup a Dask cluster composed of MIG GPUs. Please note that once a GPU is partitioned, the physical GPU (named `GPU-84fd49f2-48ad-50e8-9f2e-3bf0dfd47ccb` above) is inaccessible for CUDA compute and cannot be used as part of a Dask cluster.
+
+Alternatively, MIG instance names can be obtained programatically using [NVML](https://developer.nvidia.com/nvidia-management-library-nvml) or [PyNVML](https://github.com/gpuopenanalytics/pynvml). Please refer to the [NVML API](https://docs.nvidia.com/deploy/nvml-api/) to write appropriate utilities for that purpose.
+
+### LocalCUDACluster
+
+Suppose you have 3 MIG instances on the local system:
+
+- `MIG-41b3359c-e721-56e5-8009-12e5797ed514`
+- `MIG-65b79fff-6d3c-5490-a288-b31ec705f310`
+- `MIG-c6e2bae8-46d4-5a7e-9a68-c6cf1f680ba0`
+
+To start a `LocalCUDACluster`, the user would run the following:
+
+```python
+from dask_cuda import LocalCUDACluster
+
+cluster = LocalCUDACluster(
+    CUDA_VISIBLE_DEVICES=[
+        "MIG-41b3359c-e721-56e5-8009-12e5797ed514",
+        "MIG-65b79fff-6d3c-5490-a288-b31ec705f310",
+        "MIG-c6e2bae8-46d4-5a7e-9a68-c6cf1f680ba0",
+    ],
+    # Other `LocalCUDACluster` arguments
+)
+```
+
+### dask-cuda-worker
+
+Suppose you have 3 MIG instances on the local system:
+
+- `MIG-41b3359c-e721-56e5-8009-12e5797ed514`
+- `MIG-65b79fff-6d3c-5490-a288-b31ec705f310`
+- `MIG-c6e2bae8-46d4-5a7e-9a68-c6cf1f680ba0`
+
+To start a `dask-cuda-worker` that the address to the scheduler is located in the `scheduler.json` file, the user would run the following:
+
+```bash
+CUDA_VISIBLE_DEVICES="MIG-41b3359c-e721-56e5-8009-12e5797ed514,MIG-65b79fff-6d3c-5490-a288-b31ec705f310,MIG-c6e2bae8-46d4-5a7e-9a68-c6cf1f680ba0" dask-cuda-worker scheduler.json # --other-arguments
+```
+
+Please note that in the example above we created 3 Dask-CUDA workers on one node, for a multi-node cluster, the correct MIG names need to be specified, and they will always be different for each host.
+
+## XGBoost with Dask Cluster
+
+Currently XGBoost only exposes support for GPU communication via NCCL, which does not support MIG. For this reason, A Dask cluster that utilizes XGBoost would have to utilize TCP instead for all communications which will likely cause in considerable performance degradation. Therefore, using XGBoost with MIG is not recommended.
```

Comment:
Could we link "XGBoost" to their docs?

Reply:
Done in https://github.com/rapidsai/deployment/pull/63/commits/19c6a4169bc7ae9a262ca2188a82dedf0e3f0ff4 .